src/state/selection/logics.ts

```diff
@@ -136,8 +138,9 @@ const getIndicesForCellIds = (cellIds: string[], fullArrayOfCelIds: string[]) =>
 
 const requestCellFileInfoForSelectedArrayOfPoints = createLogic({
     process(deps: ReduxLogicDeps) {
-        const { action, imageDataSet, getState } = deps;
+        const { action, getState } = deps;
         const state = getState();
+        const imageDataSet = state.metadata.imageDataset;
```

Comment:
should always use a selector to access state

Reply:
one good reason is if you take my advice to move this to it's own branch of state, it would be a lot easier to update all of these as one single selector instead of having to search and change .metadata.imageDataset to .imageDataset.class or whatever you call the data

Reply:
Gotcha, made this change!